prometheus/src/main/scala/filodb/prometheus/parse/Parser.scala

```diff
@@ -105,6 +105,11 @@ object Parser extends StrictLogging {
     }
   }
 
+  def labelCardinalityToLogicalPlan(query: String, timeParams: TimeRangeParams): LogicalPlan = parseQuery(query) match {
+      case p: InstantExpression => p.toLabelCardinalityPlan(timeParams)
```

Comment:
Please ensure that only one ws/ns/name can be selected in the query considering performance and memory allocations when more metrics are involved.